javav2/example_code/lookoutvision/src/main/java/com/example/lookoutvision/Projects.java

```diff
@@ -0,0 +1,141 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+// snippet-start:[lookoutvision.java2.projects.complete]
+
+package com.example.lookoutvision;
+
+import software.amazon.awssdk.services.lookoutvision.LookoutVisionClient;
+import software.amazon.awssdk.services.lookoutvision.model.CreateProjectRequest;
+import software.amazon.awssdk.services.lookoutvision.model.CreateProjectResponse;
+import software.amazon.awssdk.services.lookoutvision.model.ProjectMetadata;
+import software.amazon.awssdk.services.lookoutvision.paginators.ListProjectsIterable;
+import software.amazon.awssdk.services.lookoutvision.model.ListProjectsRequest;
+import software.amazon.awssdk.services.lookoutvision.model.DeleteProjectRequest;
+import software.amazon.awssdk.services.lookoutvision.model.DeleteProjectResponse;
+import software.amazon.awssdk.services.lookoutvision.model.DescribeProjectRequest;
+import software.amazon.awssdk.services.lookoutvision.model.DescribeProjectResponse;
+import software.amazon.awssdk.services.lookoutvision.model.LookoutVisionException;
+import software.amazon.awssdk.services.lookoutvision.model.ProjectDescription;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+// Operations on Amazon Lookout for Vision projects.
+public class Projects {
+
+        public static final Logger logger = Logger.getLogger(Projects.class.getName());
+
+        /**
+         * Creates an Amazon Lookout for Vision project.
+         * 
+         * @param lfvClient   An Amazon Lookout for Vision client.
+         * @param projectName The name of the project that you want to create.
+         * @return ProjectMetadata Metadata information about the created project.
+         */
+        public static ProjectMetadata createProject(LookoutVisionClient lfvClient, String projectName)
+                        throws LookoutVisionException {
+
+                logger.log(Level.INFO, "Creating project: {0}", projectName);
+                CreateProjectRequest createProjectRequest = CreateProjectRequest.builder().projectName(projectName)
+                                .build();
+
+                CreateProjectResponse response = lfvClient.createProject(createProjectRequest);
+
+                logger.log(Level.INFO, "Project created. ARN: {0}", response.projectMetadata().projectArn());
+
+                return response.projectMetadata();
+
+        }
+
+        /**
+         * Lists the Amazon Lookout Vision projects in the current AWS account and AWS
```

Comment:
Amazon Lookout Vision --> Amazon Lookout for Vision

Reply:
Fixed.